qiskit/visualization/matplotlib.py

```diff
@@ -278,7 +272,7 @@ def draw(self, filename=None, verbose=False):
         self._get_layer_widths()
 
         # load the _qubit_dict and _clbit_dict with register info
-        n_lines = self._get_bit_labels()
+        n_lines = self._get_bit_reg_info()
```

Comment:
If we're changing the name of this function, can we make it a bit clearer that it also sets a bunch of properties on `self`?  I was confused why there was so much logic in a function that seemingly just returns the number of lines of something.

(`_set_bit_reg_info` would be better than `_get`, but there may be a better name as well)

Reply:
I like `set` in this instance. It's one of the rare functions what just sets up 2 dicts, which are then used later to do the actual drawing.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/7319/commits/65d90e68c87a048ad56dde08f2aa2a74855e215f